example/src/App.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import { ReactComponent as WordmarkAnswerBot } from "@zendeskgarden/svg-icons/src/26/answer-bot.svg";
+
+import { Widget } from "./components/Widget";
+import { Textarea } from "./components/Textarea";
+import { ConversationLog } from "./components/ConversationLog";
+import { MessageGroup } from "./components/MessageGroup";
+import { Message, Action } from "./components/Message";
+
+import styles from "./app.module.css";
+
+function App() {
```

Comment:
Can this be simplified as `const App = () => (<div>...</div>);`?

Reply:
Updated.